code/modules/griffening_cardgame/carddatums.dm

```diff
@@ -0,0 +1,685 @@
+	DEF = 25
+	Attributes = list(HUMAN, NON_ANTAG, CHANGEATTACKTARGET, COUNTER, DEATHRATTLEINCAPACITATE)
+
+/datum/griffeningcard/creature/ResearchDirector
+
+	name = "Research Director"
+	desc = "This card can only be summoned by sacrificing a scientist.. If this card is in play, increase the ATK of all scientist cards you own by 20. Once per turn at any point, all the ATK and DEF of one 'scientist' is transferred to any other card on the field. At the end of the opponents turn, this effect is reversed. "
+	LVL = 6
+	ATK = 45
+	DEF = 45
+	Attributes = list(HUMAN, NON_ANTAG, HEAD, SCIENTIST_SACRIFICE, SCIENTIST_20_ATK, SCIENTIST_STAT_DONATION)
+
+/datum/griffeningcard/creature/Scientist
+
+	name = "Scientist"
+	desc = "At the beginning of the owner of this card's turn, if there's another scientist on the field you control, increase the ATK and DEF of this card by 10 until you control no more scientists."
+	LVL = 3
+	ATK = 25
+	DEF = 25
+	Attributes = list(HUMAN, NON_ANTAG, SCIENTIST, SCIENTIST_10_STAT)
+
+/datum/griffeningcard/creature/Roboticist
+
+	name = "Roboticist"
+	desc = "If a cyborg you control is dismantled, you may summon it as soon as your turn begins."
+	LVL = 3
+	ATK = 15
+	DEF = 15
+	Attributes = list(HUMAN, NON_ANTAG, CYBORG_REVIVAL)
+
+/datum/griffeningcard/creature/ChiefEngineer
+	name = "Chief Engineer"
+	desc = "You must sacrifice a engineer to play this card. At the start of your turn, if this card is not incapacitated then you may remove the opponent's current area card. While this card is in play and the engineering area active, your area card cannot be destroyed or replaced."
+	LVL = 6
+	ATK = 40
+	DEF = 50
+	Attributes = list(HUMAN, NON_ANTAG, HEAD, ENGINEER, ENGINEER_SACRIFICE, AREA_REMOVER, AREA_REPLACER)
+
+/datum/griffeningcard/creature/Engineer
+
+	name = "Engineer"
+	desc = "At the beginning of your turn, if this card has a equipment card attached to itself, then you may search the deck for the same equipment card and put it into your hand."
+	LVL = 5
+	ATK = 25
+	DEF = 45
+	Attributes = list(HUMAN, NON_ANTAG, ONE_SACRIFICE, EQUIPMENT_GENERATOR)
+
+/datum/griffeningcard/creature/Janitor
+
+	name = "Janitor"
+	desc = "When this card is played, get a wet floor effect card from your deck and either put it on the field facedown or in your hand."
+	LVL = 2
+	ATK = 15
+	DEF = 20
+	Attributes = list(HUMAN, NON_ANTAG, SEARCH_DECK_WET_FLOOR, WET_FLOOR_REFRESHER)
+	
+/datum/griffeningcard/creature/Barman
+
+	name = "Barman"
+	desc = "As long as this card remains on the field, all non antag humans you control are immune to incapacitation."
+	LVL = 2
+	ATK = 15
+	DEF = 10
+	Attributes = list(HUMAN, NON_ANTAG, HUMAN_NONANTAG_INCAPACITATION_IMMUNITY)
+
+/datum/griffeningcard/creature/CMO
+
+	name = "Chief Medical Officer"
+	desc = "This card can only be played if you sacrifice a medical doctor. If this card is in play, increase all DEF of all humans you control by 20. This effect gives 20 more DEF per medical doctor you control."
```

Comment:
"While alive and not incapacitated, +20 DEF to all humans for each medical doctor under your control."
and make one of its types medical doctor so it counts itself, much clearer